scene/gui/tree.cpp

```diff
@@ -3455,30 +3459,48 @@ int Tree::get_item_offset(TreeItem *p_item) const {
 }
 
 void Tree::ensure_cursor_is_visible() {
-
-	if (!is_inside_tree())
+	if (!is_inside_tree()) {
 		return;
+	}
+	if (!selected_item || (selected_col == -1)) {
+		return;  // nothing under cursor
```

Comment:
Style issue around comments: https://travis-ci.org/godotengine/godot/jobs/632234552?utm_medium=notification&utm_source=github_status

BTW we're not enforcing it yet, but the current proposal for a comment style guide would be to start with a capital letter and end with a dot.